docs/release-notes/platform_10-9-0_releasenotes.md

```diff
@@ -0,0 +1,83 @@
+---
+id: v10.9.0
+title: Version 10.9.0 Release Notes
+sidebar_label: v10.9.0
+image: "img/release-note-link-preview.png"
+---
+
+_May 04, 2023_
+
+## Console
+
+### New Features
+
+#### Managing Service Accounts at Console level
+
+It is possible to create [Company-independent service accounts](TO-DO link) in case you need to give them privileges to perform [root-level operations](/development_suite/identity-and-access-management/console-levels-and-permission-management.md).
+
+:::info
+Only Console Super Users can [add](TO-DO link) or [delete](TO-DO link) Company independent service accounts, manageable exclusively at back-office level.
+:::
+
+### Bug Fix
+
+This version addressed a few bugs, here is a comprehensive list:
+
+* 
+
+## Fast Data
+
+### New Features
+
+Support to .sql file type in the DDL modal import
+
+### Improvements
+
+One to one relationships in aggregation as object instead of array.
+
+### Breaking Changes
+
+#### Revamped Single View Trigger Generator
+We're thrilled to announce the release of v.2.0.1 of the Single View Trigger Generator, one of the Fast Data component. In this release, we've given the component a complete overhaul to optimize its performance and streamline its functionality.
+
+However, as with any major update, there are a few breaking changes to be aware of. We've renamed the `STRATEGIES_FOLDER` environment variable to `MANUAL_STRATEGIES_FOLDER` and the `CUSTOM_FUNCTIONS_FOLDER` environment variable to `TRIGGER_CUSTOM_FUNCTIONS_FOLDER`. We've also removed the `USE_AUTOMATIC` variable. Additionally, the `SINGLE_VIEW_NAME`,` ER_SCHEMA_FOLDER`, and `PROJECTION_CHANGES_SCHEMA_FOLDER` variables are now required. Check out the official documentation (TODO: add link)!
+
+Do not forget to check out the [changelog](/runtime-suite/single-view-trigger-generator/changelog.md) for a full list of changes!
+
+### Bug Fix
+
+This version addressed a few bugs, here is a comprehensive list:
+
+* Projections and SoR for the no-code ERSchema still shown even if deleted
+* Single View Creator that could be attached to more Single Views
```

Comment:
@giuliowaitforitdavide let's change the perspective here, writing what is possible to be done now without describing what was not possible before

Reply:
Changed